### ISSUE TITLE
Unable to start daemon

### ISSUE THREAD

**Issue by dsemmel21** [xybu/onedrived-dev#88]
This may be related to #83, but I am not sure.  I am pretty new to Linux so...
I had originally installed onedrive-d 1.0.0.0 and it ran pretty well but decided to move to onedrived-dev as it has the ability to use ! in the ignore list.  it has worked pretty well, but I intend to run it as part of a digital picture frame, so it has to start up and run in an unattended mode.
After the installation, I ran onedrived start --debug
It asked me for my keyring password and worked just fine.

However, when I run onedrived start, it does not ask for my keyring password, and on the screen, it shows
Starting onedrived... okay, 

however, in the logs I see 
MainThread shutting down.  Code: Dying de to u$e to unhandled EOFError.

Any thoughts as to why this may not be working, or how I can get it to behave properly?

**Comment by derrix060** [comment]
This error doesn't seem to be related to the password. Can you include here the full log?

**Comment by Iito** [comment]
Same here I can't run as daemon but when I launch with --debug a password entry is prompted and it works perferctly...

**Comment by rcpangeran** [comment]
Yup, same as me, i couldn't run as daemon, but perfectly run while launch with --debug mode. Any solution for this problem?

**Comment by rcpangeran** [comment]
Tried to reconfigure everything again from beginning , then the problem gone.
Now i can use `onedrived start` without any error.

Thanks..

**Comment by Iito** [comment]
I still have the same issue even after reinstalling it and reconfigure everything.
No password prompt is made and in the log file I have this :
`Shutting down. Code: Dying due to unhandled EOFError:`

**Comment by derrix060** [comment]
@Iito can you test using my fork? I've never seen this error before....

**Comment by Tchev146** [comment]
Hi, 
I have a similar issue when i try to start the Daemon. It run well in Debug but otherwise i have this result : 

>username@localhost:~ $ NGROK=~/Documents/ngrok ~/.local/bin/onedrived start
>Starting onedrived ... OK

But in the log file, i have this result : 
>[2018-10-22 14:48:54,859] INFO: MainThread: Shutting down. Code: Dying due to unhandled EOFError: .
>[2018-10-22 14:48:54,900] INFO: MainThread: Shut down complete.

Does anyone have the solution ?

**Comment by QuentinRoyerDev** [comment]
Hi, I have the same issue, onedrive work only un debug mode. When i run the deamon, the error raised is 
"Starting onedrived ... FAILED
ERROR: Child exited immediately with exit code 127"
Any solution ?
